keps/sig-node/2411-cri-container-log-rotation/kep.yaml

```diff
@@ -0,0 +1,46 @@
+title: CRI Container Log Rotation
+kep-number: 2411
+authors:
+  - "@umohnani8"
+owning-sig: sig-node
+participating-sigs:
+  - sig-node
+  - sig-instrumentation
+  - sig-storage
+status: implemented
+creation-date: 2021-02-02
+reviewers:
+  - "@mrunalp"
+  - "@SergeyKanzhelev"
+approvers:
+  - "@mrunalp"
+  - "@SergeyKanzhelev"
```

Comment:
Approver for SIG Node will either be Dawn or Derek. If you need approvers from other SIGs you can also put them here.

Reply:
From the meeting I thought Mrunal and Sergey signed up to be both reviewers and approvers, but added Derek as well.